crates/ruff_python_formatter/tests/snapshots/format@statement__delete.py.snap

```diff
@@ -215,7 +215,8 @@ del (
 )  # Completed
 # Done
 
-del ()  # dangling end of line comment
+del (  # dangling end of line comment
```

Comment:
Can we add a test case of a empty parenthesized that has a trailing end of line and own line comments?